CMakeLists.txt

```diff
@@ -143,6 +143,9 @@ if("virtual" IN_LIST COMPILE_TARGETS)
            lua.host
            secp256k1.host
            sss.host
+           # TODO: replace with openenclave::oehostverify once OE 0.11 is
```

Comment:
Could you please open an issue for this? There's a TODO template in issues for this particular type of item.

Reply:
Obsolete, as we're not depending on `oe_verify_attestation_certificate_with_evidence` anymore and the issue was fixed in OE master.